janus_server/src/bin/aggregator.rs

```diff
@@ -58,10 +60,21 @@ struct Options {
         help = "role for this aggregator",
     )]
     role: Role,
+
     /// Password for the PostgreSQL database connection. (if not included in the connection
     /// string)
     #[structopt(long, env = "PGPASSWORD", help = "PostgreSQL password")]
     database_password: Option<String>,
+
+    /// Datastore encryption keys.
+    #[structopt(
+        long,
+        env = "DATASTORE_KEYS",
+        takes_value = true,
+        required(true),
+        help = "datastore encryption keys, encoded in base64 then comma-separated"
+    )]
+    datastore_keys: String,
```

Comment:
I think we can have clap/structopt take care of comma separation by making this a `Vec<String>`, and possibly adding `#[structopt(use_delimiter = true)]` as well. The docs for [`env()`](https://docs.rs/clap/2.34.0/clap/struct.Arg.html#method.env) say that environment variables can be used for multiple-valued arguments in such cases, and the default delimiter is the comma. (This will add an ellipsis to the argument's representation in `--help`)

Reply:
Testing quickly, your suggestion works out. 👍🏻

Reply:
Testing quickly, your suggestion works out. 👍🏻